Add Summer School Notebook to Docs (#1720)
unitaryfund/mitiq: docs/source/examples/ggi_summer_school_solved.md

---
hidden: true

> 

Author: Not sure why this is not working to fix the `toctree` error [here.](https://github.com/unitaryfund/mitiq/actions/runs/4238482095/jobs/7365600462#step:6:252) 

```
mitiq/mitiq/docs/source/examples/ggi_summer_school_solved.md: WARNING: document isn't included in any toctree
```

Author: should have used `orphan`

It is a guided tutorial on error mitigation with Mitiq and is focused on the zero-noise extrapolation (ZNE) technique. As this is
intended to be a hands-on exercise, the solutions to the examples are provided in this notebook.  

Useful links :

- [`Mitiq repository`](https://github.com/unitaryfund/mitiq)
- [`Mitiq documentation`](https://mitiq.readthedocs.io/en/stable/)
- [`Mitiq docs on ZNE`](https://mitiq.readthedocs.io/en/stable/guide/zne.html)
- [`Mitiq white paper`](https://arxiv.org/abs/2009.04417)
- [`Unitary Fund`](https://unitary.fund)

```{figure} ../img/zne_workflow2_steps.png
---
width: 400
name: figzne1
---
The diagram shows the workflow of the zero noise extrapolation (ZNE) technique in Mitiq.
```

The lab is split into the following sections :

- [](#checking-python-packages-are-installed-correctly)
- [](#computing-a-quantum-expectation-value-without-error-mitigation)
- [](#apply-zero-noise-extrapolation-with-mitiq)
- [](#explicitly-selecting-the-noise-scaling-method-and-the-extrapolation-method)
- [](#what-happens-behind-the-scenes-a-low-level-application-of-zne)
+++

## Checking Python packages are installed correctly

> This notebook was tested with **Mitiq v0.20.0** and **qiskit v0.39.0**. It probably works with other versions too. Moreover, with minor changes, it can be adapted to quantum libraries that are different from Qiskit: Cirq, Braket, PyQuil, etc..

Reviewer: ```suggestion
This notebook was tested with **Mitiq v0.23.0** and **qiskit v0.41.0**. It probably works with other versions too. Moreover, with minor changes, it can be adapted to quantum libraries that are different from Qiskit: Cirq, Braket, PyQuil, etc..
```

If you need to install Mitiq and/or Qiskit, you can uncomment and run the following cells.

```{code-cell} ipython3

> # !pip install mitiq==0.20.0

Reviewer: ```suggestion
# !pip install mitiq==0.23.0
```

```

```{code-cell} ipython3

> # !pip install qiskit==0.39.0

Reviewer: ```suggestion
# !pip install qiskit==0.41.0
```
